apps/examples/testcase/le_tc/compression/compression_tc_main.c

```diff
@@ -0,0 +1,43 @@
+/****************************************************************************
+ *
+ * Copyright 2019 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/// @file compression_tc_main.c
+
+/// @brief Main Function for Compression TestCase Example
+
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <errno.h>
+#include <fcntl.h>
+#include <tinyara/fs/fs.h>
+#include <tinyara/fs/ioctl.h>
+#include "tc_common.h"
```

Comment:
Almost headers are unnecessary for this file. Please clean up

Reply:
Done